homeassistant/components/airly/strings.json

```diff
@@ -17,7 +17,7 @@
       "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
     },
     "abort": {
-      "already_configured": "Airly integration for these coordinates is already configured."
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%] for these coordinates"
```

Comment:
We should add an `already_configured_location` common string for this.

Reply:
OK, I'll do this.